cloud-sql/postgres/knex/app.flexible.yaml

```diff
@@ -17,6 +18,7 @@ env: flex
 # The following env variables may contain sensitive information that grants
 # anyone access to your database. Do not add this file to your source control.
 env_variables:
+  INSTANCE_CONNECTION_NAME: <MY-PROJECT>:<INSTANCE-REGION>:<INSTANCE-NAME>
```

Comment:
we might actually need to leave this file as is so it doesn't mess up the quickstart: https://cloud.google.com/sql/docs/postgres/connect-instance-app-engine-flexible